pokemongo_bot/cell_workers/use_incense.py

```diff
@@ -0,0 +1,73 @@
+import time
+from pokemongo_bot.base_task import BaseTask
+from pokemongo_bot.worker_result import WorkerResult
+from pokemongo_bot.item_list import Item
+
+class UseIncense(BaseTask):
+    SUPPORTED_TASK_API_VERSION = 1
+
+    def initialize(self):
+      self.start_time = 0
+      self.use_incense = self.config.get('use_incense', False)
+      self.use_order = self.config.get('use_order', {})
+      self.incense_ordinary_count = self.bot.item_inventory_count(Item.ITEM_INCENSE_ORDINARY.value)      
```

Comment:
The way this is written right now, you will only be calculating this on startup. `initialize` is only run on bot startup`. Everything else should be done in `work`, or functions called by `work`.